proposals/lip-0067.md

```diff
@@ -46,12 +46,11 @@ We store secret recovery phrases and private keys in a JSON file following the f
 ```java
 keystoreSchema = {
   "type": "object",
-  "required": ["crypto", "metadata", "ID"],
+  "required": ["crypto", "metadata", "ID", "version"],
```

Comment:
To be easier to parse and perform potential future updates, I suggest to use the exact same order as in [ethereum specs](https://github.com/ethereum/wiki/wiki/Web3-Secret-Storage-Definition#pbkdf2-sha-256):

order: crypto, id, version, metadata.

For crypto the order is {"cipher","cipherparams",  "ciphertext", "kdf", "kdfparams", "mac"}.

Reply:
Done